packages/jose/jose.0.8.1/opam

```diff
@@ -0,0 +1,53 @@
+opam-version: "2.0"
+synopsis: "JOSE implementation for OCaml and ReasonML"
+description:
+  "JavaScript Object Signing and Encryption built ontop of pure OCaml libs"
+maintainer: ["ulrik.strid@outlook.com"]
+authors: ["Ulrik Strid"]
+license: "MIT"
+homepage: "https://ulrikstrid.github.io/reason-jose"
+doc: "https://ulrikstrid.github.io/reason-jose"
+bug-reports: "https://github.com/ulrikstrid/reason-jose/issues"
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "base64" {>= "3.0.0"}
```

Comment:
```suggestion
  "base64" {>= "3.3.0"}
```
Otherwise the variant returned by `Base64.decode` is not open:
```

#=== ERROR while compiling jose.0.8.1 =========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.5.0.0 | pinned(https://github.com/ulrikstrid/reason-jose/releases/download/v0.8.1/jose-v0.8.1.tbz)
# path                 ~/.opam/5.0/.opam-switch/build/jose.0.8.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p jose -j 71 @install
# exit-code            1
# env-file             ~/.opam/log/jose-7-95ffee.env
# output-file          ~/.opam/log/jose-7-95ffee.out
### output ###
# (cd _build/default && /home/opam/.opam/5.0/bin/ocamlc.opt -w -40 -g -bin-annot -I jose/.Jose.objs/byte -I /home/opam/.opam/5.0/lib/asn1-combinators -I /home/opam/.opam/5.0/lib/astring -I /home/opam/.opam/5.0/lib/base/caml -I /home/opam/.opam/5.0/lib/base64 -I /home/opam/.opam/5.0/lib/bigarray-compat -I /home/opam/.opam/5.0/lib/biniou -I /home/opam/.opam/5.0/lib/camlp-streams -I /home/opam/.opam/5.0/lib/cstruct -I /home/opam/.opam/5.0/lib/domain-name -I /home/opam/.opam/5.0/lib/easy-format -I /home/opam/.opam/5.0/lib/eqaf -I /home/opam/.opam/5.0/lib/eqaf/bigstring -I /home/opam/.opam/5.0/lib/eqaf/cstruct -I /home/opam/.opam/5.0/lib/fmt -I /home/opam/.opam/5.0/lib/gmap -I /home/opam/.opam/5.0/lib/logs -I /home/opam/.opam/5.0/lib/mirage-crypto -I /home/opam/.opam/5.0/lib/mirage-crypto-ec -I /home/opam/.opam/5.0/lib/mirage-crypto-pk -I /home/opam/.opam/5.0/lib/mirage-crypto-rng -I /home/opam/.opam/5.0/lib/nocrypto -I /home/opam/.opam/5.0/lib/parsexp -I /home/opam/.opam/5.0/lib/pbkdf -I /home/opam/.opam/5.0/lib/ppx_deriving/runtime -I /home/opam/.opam/5.0/lib/ppx_sexp_conv/runtime-lib -I /home/opam/.opam/5.0/lib/ptime -I /home/opam/.opam/5.0/lib/result -I /home/opam/.opam/5.0/lib/rresult -I /home/opam/.opam/5.0/lib/sexplib -I /home/opam/.opam/5.0/lib/sexplib0 -I /home/opam/.opam/5.0/lib/x509 -I /home/opam/.opam/5.0/lib/yojson -I /home/opam/.opam/5.0/lib/zarith -no-alias-deps -open Jose__ -o jose/.Jose.objs/byte/jose__Jwk.cmo -c -impl jose/Jwk.ml)
# File "jose/Jwk.ml", line 496, characters 39-61:
# 496 |   with Json.Type_error (s, _) -> Error (`Json_parse_failed s)
#                                              ^^^^^^^^^^^^^^^^^^^^^^
# Error: This expression has type [> `Json_parse_failed of string ]
#        but an expression was expected of type [ `Msg of string ]
#        The second variant type does not allow tag(s) `Json_parse_failed
```